GOVERNANCE.md

```diff
@@ -131,3 +122,75 @@ the voting process should be considered.
 
 Additions and removals of maintainers require a **2/3 majority**, while other decisions and changes
 require only a simple majority.
+
+## Change Approval
```

Comment:
I split out the change approval process from the governance, the only update to the governance now is the process for adding/removing members to the change approval roles.

Note that I described the change approval process in CONTRIBUTING.md, it makes a lot more sense there to me since it describes a key element of the contributing process.